src/styles/blocks/calendar.scss

```
@@ -0,0 +1,57 @@
+body {
+      display: flex;
+      justify-content: center;
+      text-align: center;
+      margin: 0;
+      font-family: Arial, Helvetica, sans-serif;
+      min-height: 100vh;
+      align-items: center;
+    }
```

```suggestion
body {
  display: flex;
  justify-content: center;
  text-align: center;
  margin: 0;
  font-family: Arial, Helvetica, sans-serif;
  min-height: 100vh;
  align-items: center;
}
```
Consider fixing indents according to the code style guide

---

It also would be better to move some general page styles to a separate file